### ISSUE TITLE
"Filter unblocking search ads and self-promotions" filter does not work.

### ISSUE THREAD

**Issue by PentagramTr** [AdguardTeam/AdguardForiOS#1055]
I've enabled this filter, but I can't see any sponsored ads.If I tap the "Reload without Content Blockers" button, sponsored ads appears.

<details><summary>Filter:</summary>

![IMG_5676](https://user-images.githubusercontent.com/34379403/59217097-e0551000-8bc5-11e9-92d0-6304fdfb14e8.jpg)

</details>

### Steps to reproduce

1.
2.
3.
4.

### Expected behavior
Sponsored ads should appear
<details><summary>Screenshot:</summary>

![PICSEW_9003](https://user-images.githubusercontent.com/34379403/59217016-b26fcb80-8bc5-11e9-905c-6445078b60eb.png)

</details>

### Actual behavior

<details><summary>Screenshot:</summary>

![PICSEW_F4A4](https://user-images.githubusercontent.com/34379403/59216726-eac2da00-8bc4-11e9-8a40-3d24b423db28.png)

</details>

### Your environment

* Device model and storage size: iPhone 5s 16GB
* Operating system and version: 12.3.1
* Browser Safari (Engine:Chrome)
* Wi-fi
* AdGuard Premium

**Comment by ameshkov** [comment]
Uh, got it. The problem is that the filters from "Other" group are used by a different content blocker (the one marked as "Other") and have no effect on the "General" group.

This is a complicated issue which we need to solve in one of the future updates.

We should allow changing content blockers "affinity" on the filter lists level or even on the individual rules level.

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdguardForiOS/issues/1104
